addon/chrome/content/preferences/subdialogs/pref-appearance-ce.js

```diff
@@ -254,11 +254,11 @@
       <spacer flex="1" class="visible"></spacer>
       <html:input anonid="color" class="visible" palettename="standard" type="color" inherits="disabled"></html:input>
       <label value="[RGB]:" inherits="disabled"></label>
-      <html:input anonid="red" class="rgbcontrol" inherits="disabled" maxlength="3" size="5" type="number" min="0" max="255"></html:input>
-      <html:input anonid="green" class="rgbcontrol" inherits="disabled" maxlength="3" size="5" type="number" min="0" max="255"></html:input>
-      <html:input anonid="blue" class="rgbcontrol" inherits="disabled" maxlength="3" size="5" type="number" min="0" max="255"></html:input>
+      <html:input anonid="red" class="rgbcontrol" inherits="disabled" maxlength="3" size="5" type="number" required="" min="0" max="255"></html:input>
```

Comment:
Use only **```required```** without value no need add required=""

Reply:
Last time I did that it throw me a parser error.

Reply:
You are right, so set it to ```required="required"```, that Firefox developer does.